src/app/components/apps-view/apps-view.component.html

```diff
@@ -17,6 +17,25 @@
  -->
 
 <div class="apps-view">
+  <mat-form-field class="search-wrapper white-mat-form-field">
```

Comment:
@akhilpb001 cud we also have a filter on left side with queue name, hence search will be very easy?

Reply:
Do you mean two input boxes side by side, one for appId search and another for queueName?

Reply:
Yes,That will be a good start if we cant get a filter check box.
if filters is available, then we can skip this for now

Reply:
Usually search will be applied on whole table and its not good ux if we have checkbox for selecting column to be searched.

Reply:
I agree. I remember some old Ui where each column search were there, In YARN UI2, we have a regex based search. Is it similar here?

Reply:
Nope. Here we have one input box where you can search for either appId or queueName. It is a string contains search. As you start search, it will shows the rows that contains the search text either in appId or queueName. Since there wont be much overlapping of text, so the probability of accurate search is high.

Reply:
ok. i think its fine for now